### PR TITLE
Update lab11.md

### PR DESCRIPTION
**Pull request by atroxix**
Hint for the environment variable that GitHub sends the secret with - issue #93

Signed-off-by: Silver Salonen <silver.salonen@gmail.com>

### CODE REVIEW

**Comment by ivanayov** [comment]
Thank you for your contribution @atroxix 

Can you sign your commit with 
```
git commit --amend -s
```
and then `push -f`?

Can you also update the PR description with a reference to the issue it covers?

File: lab11.md [hunk 1]
```diff
@@ -189,6 +189,6 @@ $ echo -n "This is a message" | faas-cli invoke hmac-protected --sign hmac --key
 HMAC validation failed.
 ```
 
-As a follow-up task you could apply HMAC to secure your endpoint on `issue-bot` from [lab 5](https://github.com/openfaas/workshop/blob/7f1b0246376b7e4380bb5874655f28254c15f749/lab5.md)
+As a follow-up task you could apply HMAC to secure your endpoint on `issue-bot` from [lab 5](https://github.com/openfaas/workshop/blob/7f1b0246376b7e4380bb5874655f28254c15f749/lab5.md) (pay attention to the environment variable used by GitHub to pass the secret value).
```
Comment (alexellis) [inline lab11.md]: @rgee0 do we pass secrets over environmental variables?

Comment (rgee0) [reply lab11.md]: I read this to be referring to the the incoming GitHub message's `X-Hub-Signature` value which is passed through to a function as an env-var.

I think this needs to be more specific a message in order for it to be helpful for workshop attendees.

Comment (atroxix) [reply lab11.md]: My idea was not to specifically tell what needs to be done, but rather provide a hint to give some direction to somebody who may get stuck behind this.

But of course it's up to you whether and how you want to have that hint there.

**Comment by atroxix** [comment]
> Can you sign your commit with
> 
> ```
> git commit --amend -s
> ```
> 
> and then `push -f`?
> 
> Can you also update the PR description with a reference to the issue it covers?

I now updated the description. Do I understand correctly that in order to sign my commit, I need to first checkout the repository and then push from my computer? I now did it via GitHub only, so getting it to my computer first seems like a bit of waste.

File: lab11.md (hunk 1 above)
Comment (rgee0) [reply lab11.md]: The point is more that we should be precise with the terminology.

If it **is** a secret then we would treat it as such, and not give the impression we use env vars for storing secrets.

If it **isn't** a secret then lets not refer to it as a secret.

I think that here we are probably passing a hashed value from the HTTP headers as an environment variable in order to verify that the message hasn't been altered.

**Comment by rgee0** [comment]
@atroxix you can still sign off through the GUI, you just need to add the following to your commit message:

`Signed-off-by: Your Name <your@email.com>`

It might mean closing this PR and opening a new one, however.

File: lab11.md (hunk 1 above)
Comment (atroxix) [reply lab11.md]: Hmm, somehow it seemed to me that it was really a secret. Although I don't remember anymore what exactly did I do there - quite some time has passed from that.

**Comment by alexellis** [comment]
Derek close: inactive

**Pull request closed without merging**